### DISCUSSION TITLE
I can help you with the RO translation

### DISCUSSION THREAD

**Discussion by vlad-cotfas** [rxhanson/Middle-Localization#41] [General]
I can help you with the RO translation 

Todo:
https://github.com/rxhanson/Middle-Community/blob/master/localization/ro.lproj/Main.strings

I already bought your trackpad app and planning to have the Rectangle Pro as well, I might need Rectangle license 💯

**Comment by vlad-cotfas** [comment]
Especially on how you throw any window from one monitor to another, not sure how you managed to do that but I want to have it in C++ code. (i know is offtopic!)
But if I do not manage to implement such a thing I will have to use Rectangle Pro and with Karabiner and hotkeys.

**Comment by rxhanson** [comment]
Thanks for the kind words on Rectangle Pro. Sorry for the delay on getting back to you here. Middle translations are only good for Middle license keys, as I have had some people try to take advantage of my offer with multiple minor contributions. I am planning on localizing my other apps and will have similar contribution incentives for each one, when the time comes.
